cmd/kogito/command/shared/operatorhub.go

```diff
@@ -81,20 +80,21 @@ func installOperatorWithOperatorHub(namespace string, cli *client.Client) error
 
 	subscription := &olmapiv1alpha1.Subscription{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      defaultOperatorPackageName,
+			Name:      DefaultOperatorPackageName,
 			Namespace: namespace,
 		},
 		Spec: &olmapiv1alpha1.SubscriptionSpec{
-			Package:                defaultOperatorPackageName,
-			CatalogSource:          communityOperatorSource,
-			CatalogSourceNamespace: operatorMarketplaceNamespace,
-			Channel:                defaultOperatorChannelName,
+			Package:                DefaultOperatorPackageName,
+			CatalogSource:          CommunityOperatorSource,
+			CatalogSourceNamespace: OperatorMarketplaceNamespace,
+			Channel:                channel.String(),
```

Comment:
you could just use `fmt.Sprintf("%v", channel)`

Reply:
cast channel direct to string